flink-connector/flink/src/test/java/org/apache/gravitino/flink/connector/integration/test/hive/FlinkHiveCatalogIT.java

```diff
@@ -156,9 +157,10 @@ public void testCreateGravitinoHiveCatalog() {
     tableEnv.useCatalog(DEFAULT_CATALOG);
     tableEnv.executeSql("drop catalog " + catalogName);
     Assertions.assertFalse(metalake.catalogExists(catalogName));
-
-    Optional<Catalog> droppedCatalog = tableEnv.getCatalog(catalogName);
-    Assertions.assertFalse(droppedCatalog.isPresent(), "Catalog should be dropped");
+    Assertions.assertThrows(
```

Comment:
According to the API of `CatalogStore`,  is it proper to throw CatalogException when catalog is not exists? 

```
    /**
     * Get a catalog by name.
     *
     * @param catalogName name of the catalog to retrieve
     * @return the requested catalog or empty if the catalog does not exist
     * @throws CatalogException in case of any runtime exception
     */
    Optional<CatalogDescriptor> getCatalog(String catalogName) throws CatalogException;

```

Reply:
fixed